config/firstboot/foot.ini

```diff
@@ -0,0 +1,227 @@
+# -*- conf -*-
+
+# shell=$SHELL (if set, otherwise user's default shell from /etc/passwd)
+# term=foot (or xterm-256color if built with -Dterminfo=disabled)
+term=xterm-256color
```

Comment:
I am on the edge with this, constantly switching between switching it off and on. Yes, this is more robust and works better if it leaks to other machines which have no idea about `foot`. On the other hand, a non-zero number of applications (e.g., `xdg-terminal`, which, for better or worse, we have in openSUSE) expect that `TERM` contains a name of the executable to run new terminal (and there are some historical reasons why it is not completely crazy, it may be even POSIX). Also, `rofi-sensible-terminal` needs `TERMINAL` variable (exactly because of the confusing state of `TERM`), because [it doesn’t know about `foot` otherwise](https://github.com/lbonn/rofi/blob/wayland/script/rofi-sensible-terminal). It is a mess.